apps/portals/src/portal-components/csbc-home-page/style/_ecosystem.scss

```diff
@@ -17,46 +17,46 @@
     margin-bottom: 20px;
   }
   .control-container {
-    background: white;
     display: flex;
-    border: 1px solid #dcdcdc;
```

Comment:
Border and background were useful for the cancer complexity portal, where the component is not shown on a white background. Think it still looks ok: 

<img width="1216" alt="image" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/aad33679-9a59-4202-af78-98a786c04cdc">

Reply:
I agree